docker_plugin/tests/blueprint/test_pull.yaml

```diff
@@ -38,6 +38,13 @@ node_templates:
               stdin_open: true
               tty: true
               command: /bin/sleep 30
+            daemon_client:
```

Comment:
If the plugin yaml specifies the default values, why do you need to put them in the blueprint if they are not overriding any particular values?